homework_5/demo/src/main/java/com/epam/demo/SpringBootTestProjectApplication.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.demo;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication(scanBasePackages = "com.epam.*")
```

Comment:
If you have your class annotated with `@SpringBootApplication` in the root package you don't have to specify `scanBasePackages`